Chatkit/Repositories/JoinedRoomsRepository.swift

```diff
@@ -67,7 +93,7 @@ public class JoinedRoomsRepository: JoinedRoomsRepositoryProtocol {
         self.versionedState = buffer.currentState
         self.connectionState = connectivityMonitor.connectionState
         
-        self.state = JoinedRoomsRepository.state(forConnectionState: connectivityMonitor.connectionState,
+        self.state = Self.state(forConnectionState: connectivityMonitor.connectionState,
                                                  versionedState: buffer.currentState,
                                                  previousVersionedState: nil,
                                                  usingTransformer: dependencies.transformer)
```

Comment:
Really no big deal but there's a slight whitespace/padding mis-alignment as a result of your change.

Reply:
I will fix that.